internal/db/init.go

```diff
@@ -2,13 +2,20 @@ package db
 
 import (
 	"context"
+	"log"
+	"os"
 
 	"go.mongodb.org/mongo-driver/mongo"
 	"go.mongodb.org/mongo-driver/mongo/options"
 )
 
 func Initialize(ctx context.Context) (*mongo.Client, error) {
-	clientOpts := options.Client().ApplyURI("mongodb://root:example@localhost:27017")
+	mongoURI := os.Getenv("MONGO_URL")
```

Comment:
This may break local development unless that variable is set. We likely want to write some docs to testing / running the application locally using docker. I created this issue for that: https://github.com/hay-i/gooal/issues/86